#### Bayesian Statistics Hands-On
For this hands-on, you will be determining which type of mold-removal solution works better: just bleaching objects, or bleaching them and scrubbing them down thoroughly out of 10,000 trials. Based on the priors you created, the mold-removal solutions have a 90% chance of working.

You're trying to determine whether the mold will grow back or not

In [1]:
trials <- 10000

#### alpha is the 90% chance of working that we want. By default, 10 % is the beta - the outcome we don't want.

In [2]:
alpha <- 9
beta <- 1

#### set up rbeta()

In [3]:
samplesA <- rbeta(trials, 27+alpha, 39 + beta)
samplesB <- rbeta(trials, 10+alpha, 45 + beta)

#### samplesA calculates the probability of bleach working. samplesB calculates the probability it doesn't work

In [4]:
Bsuperior <- sum(samplesB > samplesA) / trials

In [5]:
print(Bsuperior)

[1] 0.0123


# there's a .0123 % chance the bleach is effective.  That doesn't seem like much? 